components/style_derive/to_css.rs

```diff
@@ -150,6 +150,20 @@ fn derive_single_field_expr(
     where_clause: &mut WhereClause,
 ) -> Tokens {
     if attrs.iterable {
+        if let Some(if_empty) = attrs.if_empty {
+            return quote! {
+                {
+                    let mut iter = #field.iter().peekable();
```

Comment:
There's no need to use `peekable`, right? You could just match on the first element returned and such... Not a big deal I guess.